src/Service/Logger.service.js

```diff
@@ -16,6 +16,12 @@ if (sentryIsAvailable) {
   });
 }
 
+export const LOGGING_LEVELS = {
```

Comment:
I would perhaps call this `WARNING_LEVELS` as it's specific to the warning method and nothing else

Reply:
e421f27e8d45b1dc2d6c3ad5925bd281edd51d7d